python/cuml/metrics/cluster/adjustedrandindex.pyx

```diff
@@ -0,0 +1,87 @@
+#
+# Copyright (c) 2019, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# cython: profile=False
+# distutils: language = c++
+# cython: embedsignature = True
+# cython: language_level = 3
+
+import numpy as np
+
+from libc.stdint cimport uintptr_t
+
+from cuml.common.handle cimport cumlHandle
+from cuml.utils import input_to_dev_array
+import cuml.common.handle
+cimport cuml.common.cuda
+
+cdef extern from "metrics/metrics.hpp" namespace "ML::Metrics":
+
+    double adjustedRandIndex(cumlHandle &handle,
+                             int *y,
+                             int *y_hat,
+                             int n,
+                             int lower_class_range,
+                             int upper_class_range)
+
+
+def adjusted_rand_score(labels_true,
+                        labels_pred,
+                        handle=None):
+    """
+    Adjusted_rand_score function is a modified rand index
+    function modified to handle chance.
+
+        Parameters
+        ----------
+            labels_true : Array of response variables of
+                              the first clustering classifications
+            labels_pred : Array of response variables of
+                              the first clustering classifications
+            handle : cuml.Handle
+        Returns
+        -------
+            The adjusted rand index value
+    """
+
+    handle = cuml.common.handle.Handle() \
+        if handle is None else handle
+    cdef cumlHandle* handle_ =\
+        <cumlHandle*><size_t>handle.getHandle()
+    labels_true = labels_true.astype(np.int32)
```

Comment:
There should be a warning informing that a conversion is happening if the input is not `int32`